### DISCUSSION TITLE
zh/guides/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#12787] [Docs]
# zh/guides/

深入探讨Ultralytics'YOLO 。了解YOLO 物体检测模型、如何在自定义数据上进行训练、多 GPU 训练、导出、预测、部署等。

https://docs.ultralytics.com/zh/guides/

**Comment by thisisqiuqiuqiu** [comment]
I would like to ask, does the yolov8 base model need to be trained to 500 epochs？

**Reply by glenn-jocher** [reply]
Hello! The number of epochs needed for training the YOLOv8 base model can vary depending on your specific dataset and the complexity of the task. While 500 epochs might be necessary for some scenarios to achieve optimal results, it's not a strict requirement for all cases.

A good practice is to monitor the training process for signs of convergence, such as stable validation loss and accuracy metrics. You can start with fewer epochs, like 100 or 200, and evaluate the model's performance. If the results are not satisfactory, you can continue training for more epochs or consider adjusting other hyperparameters.

Here's a simple example to start training with a specified number of epochs:

```bash
yolo train data=coco128.yaml model=yolov8n.pt epochs=100
```

Adjust the `epochs` value as needed based on your observations. Happy training! 😊

**Comment by thisisqiuqiuqiu** [comment]
Thank you very much for your letter. Your reply is very helpful to my experimental training. I understand how to set up the basic model training. I wish you all success in your future work.&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ultralytics/ultralytics"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月20日(星期一) 中午12:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ultralytics/ultralytics] zh/guides/ (Discussion #12787)





 
Hello! The number of epochs needed for training the YOLOv8 base model can vary depending on your specific dataset and the complexity of the task. While 500 epochs might be necessary for some scenarios to achieve optimal results, it's not a strict requirement for all cases.
 
A good practice is to monitor the training process for signs of convergence, such as stable validation loss and accuracy metrics. You can start with fewer epochs, like 100 or 200, and evaluate the model's performance. If the results are not satisfactory, you can continue training for more epochs or consider adjusting other hyperparameters.
 
Here's a simple example to start training with a specified number of epochs:
 yolo train data=coco128.yaml model=yolov8n.pt epochs=100
 
Adjust the epochs value as needed based on your observations. Happy training! 😊

**Reply by glenn-jocher** [reply]
@thisisqiuqiuqiu thank you for your kind words! I'm glad to hear that the information was helpful for your experimental training. If you have any more questions or need further assistance as you continue with your projects, feel free to reach out. Wishing you great success in all your endeavors! 😊
